org.eclipse.jdt.core.tests.compiler/src/org/eclipse/jdt/core/tests/compiler/regression/MethodVerifyTest.java

```diff
@@ -12391,19 +12385,25 @@ public void test331446() {
 			"}\n" +
 			"",
 		},
-		"",
+		"""
+		----------
+		1. ERROR in X.java (at line 12)
+			Test.assertEquals("Test", comparator, new ArrayList(), new ArrayList());
+			     ^^^^^^^^^^^^
+		The method assertEquals(String, Comparator<Object>, List<Object>, List<Object>) is ambiguous for the type Test
+		----------
```

Comment:
As per bug 331446 the original problem can only be reproduces with a mix of 1.4 & 1.5.

I vote for removing.

Details: at 1.4 we could not see the generic signature of `assertEquals(String, Comparator<T>, T, T)`. Based on erasures the other method, `assertEquals(String, Comparator, List, List)` is more specific and therefor it could be selected. Since 1.5 neither method is more specific than the other wrt the invocation.

Reply:
Will remove.